backoffice/src/Models/FormModels/WreckMaterialForm.cs

```diff
@@ -34,6 +35,7 @@ public WreckMaterialForm(WreckMaterial wreckMaterial)
     public Guid DroitId { get; set; }
 
     public string Name { get; set; } = string.Empty;
+    public AddressForm Address { get; set; } = new();
```

Comment:
Can we rename this StorageAddress pls?
public Address StorageAddress { get; set; } = new();